src/base/memory_monitor_sr.cpp

```diff
@@ -30,6 +30,11 @@
 #include "memory_monitor_sr.hpp"
 #include "log_impl.h"
 
+#if !defined (NDEBUG)
+thread_local char mmon_err_msg[DB_MAX_IDENTIFIER_LENGTH];
```

Comment:
Please elaborate on why this must be thread_local.

Reply:
I devided it to another issue.